src/button/_button.scss

```diff
@@ -31,8 +31,20 @@
   cursor: not-allowed;
 }
 
+.button--is-pending {
+  color: var(--button-bg);
+}
```

Comment:
I have read the conversation above, I think this may cause strange bugs. How about setting `opacity: 0`? Then we can make sure it is not visible.

Also, you said spinner wasn't working for the Ediz's method. Maybe it can be because `.button` is not relatively positioned (see my other comment)

Reply:
I've tried again but unfortunately, the result is the same. Also, when we set `opacitiy: 0`, the wrapper is disappearing with the background and all of them.

I guess we need to declare to children, not button classes for `opacity` or `visibility` properties.